app/services/proofing/lexis_nexis/request_signer.rb

```diff
@@ -0,0 +1,43 @@
+module Proofing
+  module LexisNexis
+    class RequestSigner
+      attr_accessor :config, :message_body, :path
+
+      # @param config must respond to base_url, hmac_key_id, and hmac_secret_key
+      def initialize(config:, message_body:, path:)
+        @config = config
+        @message_body = message_body
+        @path = path
+      end
+
+      # Example HMAC auth header from RDP_REST_V3_DecisioningGuide_March22.pdf, page 21
```

Comment:
I feel like "Example" is misleading here?
```suggestion
      # HMAC auth header from RDP_REST_V3_DecisioningGuide_March22.pdf, page 21
```